feat(mesh): Mesh/Konnect docs revamp (#6512)
Kong/developer.konghq.com: app/_how-tos/mesh/migrate-self-managed-zone-to-konnect.md

      include_content: cleanup/mesh/remove-working-directory
faqs:
  - q: How does a zone authenticate to the global control plane?
    a: |
      {% include faqs/mesh-zone-authentication.md %}
---

If you already run {{site.mesh_product_name}} with a self-managed global control plane, you can migrate your zones to a {{site.konnect_short_name}}-managed global control plane so that {{site.konnect_short_name}} hosts and operates the global control plane for you. To learn more, see [{{site.mesh_product_name}} in {{site.konnect_short_name}}](/mesh/konnect/).

This guide migrates a self-managed zone control plane to {{site.konnect_short_name}} by transferring your meshes and policies to the {{site.konnect_short_name}}-managed global control plane and reconnecting the zone.

{:.warning}
> This process assumes you're migrating zones one by one. During migration, zone-to-zone communication may break temporarily because each zone's zone ingress must be registered with the new global control plane in {{site.konnect_short_name}}. Until both zones are migrated, cross-zone service discovery won't work.

## Transfer resources from the self-managed global control plane

> 1. Create and enter a working directory for this guide:

Reviewer: ```suggestion
1. Create and navigate to the working directory for this guide:
```
